source/wp-content/themes/wporg-developer-2023/theme.json

```diff
@@ -13,36 +13,54 @@
 					"slug": "linen",
 					"color": "#fbfaf7",
 					"name": "Linen"
-
 				},
 				{
 					"slug": "syntax-black",
-					"color": "#000000",
-					"name": "Black"
+					"color": "var(--wp--preset--color--black)",
+					"name": "Syntax Black"
 				},
 				{
 					"slug": "syntax-red",
-					"color": "#d63638",
-					"name": "Red"
+					"color": "var(--wp--custom--color--red-50)",
+					"name": "Syntax Red"
 				},
 				{
 					"slug": "syntax-green",
-					"color": "#008a20",
-					"name": "Green"
+					"color": "var(--wp--custom--color--green-50)",
+					"name": "Syntax Green"
 				},
 				{
 					"slug": "syntax-blue",
-					"color": "#135e96",
-					"name": "Blue"
+					"color": "var(--wp--custom--color--blue-60)",
+					"name": "Syntax Blue"
 				},
 				{
 					"slug": "syntax-grey",
-					"color": "#646970",
-					"name": "Grey"
+					"color": "var(--wp--custom--color--grey-50)",
+					"name": "Syntax Grey"
+				},
+				{
+					"slug": "syntax-yellow",
+					"color": "var(--wp--custom--color--yellow-50)",
+					"name": "Syntax Yellow"
 				}
 			]
 		},
 		"custom": {
+			"color": {
```

Comment:
Colors picked from [existing sass palette](https://github.com/WordPress/wporg-developer/blob/trunk/source/wp-content/themes/wporg-developer-2023/scss/settings/_colors.scss) which is unused in the new theme.

Adding to custom means they are available for semantic colors in blocks but don't show in the editor palette.